Logger saved at ./logs/OTB100/siamrpn_mobilev2_l234_dwxcorr/54/4_net_G.pth/log_22:53:48.txt
Ran experiment with command: "python ttattack_untargeted.py --tracker_name=siamrpn_mobilev2_l234_dwxcorr --dataset=OTB100 --case=54 --gpu=1 --model_iter=4_net_G.pth --attack_universal"
